libethereum/BlockChain.cpp

```diff
@@ -84,17 +84,23 @@ std::ostream& dev::eth::operator<<(std::ostream& _out, BlockChain const& _bc)
 
 ldb::Slice dev::eth::toSlice(h256 const& _h, unsigned _sub)
 {
-#if ALL_COMPILERS_ARE_CPP11_COMPLIANT
```

Comment:
any reason to remove this? the potential code allowed through c++11 once we target the newer MSVC is substantially cleaner.